syntax = "proto3";
package task;

service TaskService {
    rpc CreateTask(CreateTaskRequest) returns (TaskResponse);
    rpc GetTask(GetTaskRequest) returns (TaskResponse);
}

message CreateTaskRequest {
    string title = 1;
    string description = 2;
    string user_id = 3;
}

message TaskResponse {
    string task_id = 1;
    string title = 2;
    string description = 3;
    string user_id = 4;
}

message GetTaskRequest {
    string task_id = 1;
}
